src/alias.ml

```diff
@@ -83,7 +83,7 @@ let standard_aliases = Hashtbl.create 7
 let is_standard name = Hashtbl.mem standard_aliases name
 
 let make_standard name =
-  Hashtbl.add standard_aliases name ();
+  Hashtbl.add_exn standard_aliases name ();
```

Comment:
`set` seems better here.

Reply:
Why? it shouldn't be possible to override an alias after it's been set.

Reply:
It's OK to do that though: the operation is idempotent. Well, either way seems fine.

Reply:
It's ok, but it's still probably unintended.